.circleci/config.yml

```diff
@@ -249,12 +249,12 @@ jobs:
                     cp src/app/libp2p_helper/result/bin/libp2p_helper package/coda-libp2p_helper
                     chmod +w package/coda-libp2p_helper
                     ./scripts/librewrite-macos.sh package/coda-libp2p_helper
-                    cp src/_build/default/app/logproc/logproc.exe package/coda-logproc
+                    cp _build/default/app/logproc/logproc.exe package/coda-logproc
```

Comment:
Should this not be `cp _build/default/src` if we're building from root? Ditto below